src/uvw/util.hpp

```diff
@@ -314,7 +320,7 @@ Addr address(F &&f, const H *handle) noexcept {
 template<typename F, typename... Args>
 std::string path(F &&f, Args... args) noexcept {
     std::size_t size = DEFAULT_SIZE;
-    char buf[size];
+    char buf[DEFAULT_SIZE];
```

Comment:
Is it a MSVC problem? The current code uses `char buf[size]` and compiles fine. Anyway, if it doesn't work, let's use `DEFAULT_SIZE` to initialize `size` and that's all. Much better than `std::size_t size = sizeof(buf);`. :-)